optuna/trial.py

```diff
@@ -148,6 +160,82 @@ def _init_relative_params(self):
             self.study, trial, self.relative_search_space
         )
 
+    @experimental("1.2.0")
+    def suggest_float(self, name, low, high, *, log=False):
+        # type: (str, float, float, bool) -> float
+        """Wrapper method for ``suggest_uniform`` and ``suggest_loguniform``.
```

Comment:
Personally, at the first line, I prefer writing a sentence that briefly describes the behavior of the target method (see below as an example): 
```
Suggest a value for the floating point parameter.

Note that this is a wrapper method for :func:`~optuna.trial.Trial.suggest_uniform` and  :func:`~optuna.trial.Trial.suggest_loguniform`.
```
However, this is not a strong opinion and the current doc seems good as well.

Reply:
Thank you for the suggestion and I agree with your idea.
Let me apply your feedback. (6ff1d40)